CRM/Upgrade/Incremental/php/FiveTwo.php

```diff
@@ -61,6 +61,32 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
     // }
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_2_alpha1($rev) {
```

Comment:
@eileenmcnaughton like I was saying if we need to set it to ```upgrade_5_2_alpha1```

Reply:
oh - will watch & see if @totten / @colemanw merge this :-)